source/dub/project.d

```diff
@@ -1756,6 +1756,7 @@ final class SelectedVersions {
 	}
 
 	/// Selects a certain path for a specific package.
+	deprecated("Selecting path dependency is deprecated")
```

Comment:
should these be deprecated? If we add `dub select [dependency] [path]`, it would call this function

Reply:
It would only make sense if the dependency was `optional`, correct ?

Reply:
also if you locally develop with a different test version you have locally checked out in your project, but don't want to register it globally across all your projects

Reply:
I don't follow ? If you develop locally, you just change your `dub.json`, don't you ?

Reply:
you are right, it's probably better to put it inside `dub.json`/`dub.sdl` then